src/transformers/integrations/flash_attention.py

```diff
@@ -18,8 +21,29 @@ def flash_attention_forward(
     scaling: Optional[float] = None,
     sliding_window: Optional[int] = None,
     softcap: Optional[float] = None,
+    output_attentions: bool = False,
+    head_mask: Optional[torch.Tensor] = None,
+    eager_fallback: Optional[Callable] = None,
     **kwargs,
 ) -> Tuple[torch.Tensor, None]:
+    if output_attentions or head_mask is not None:
+        logger.warning_once(
+            "Falling back to eager attention because `flash_attention_2` does not support"
+            " `output_attentions=True` or `head_mask`."
+        )
+        return eager_fallback(
```

Comment:
if eager fallback is none we are failing hard! Let's maybe just emit the warning no? And return None output attentions